pages/index.js

```diff
@@ -1,64 +1,23 @@
 import styled from "styled-components";
 import { useState } from "react";
-import Link from "next/link";
 
-const Header = styled.header`
-  font-family: monda;
-  font-size: 2em;
-  color: #e7e4d8;
-  font-weight: bold;
-  text-align: center;
-  background-color: #78c9cc;
-`;
-
-const Form = styled.form`
-  color: black;
-  display: grid;
-  width: 70%;
-  margin: 5px auto;
-  padding: 10px;
-  border-radius: 5px;
-  text-align: center;
+const Fieldset = styled.fieldset`
```

Comment:
maybe you could name it StyledFieldset .not nessesary but maybe better consistens in your code